### PR TITLE
Create config file for openstack cloud provider

### PR DESCRIPTION
**Pull request by ss22ever**
#479

### CODE REVIEW

**Comment by dims** [comment]
/lgtm

Let's do this!

File: phase1/openstack/openstack.jsonnet [hunk 1]
```diff
@@ -69,6 +69,17 @@ function(cfg)
             k8s_master_ip: "${openstack_compute_instance_v2.%(master_instance)s.network.0.fixed_ip_v4}" % names,
           },
         },
+       	//Populates openstack config file
+        cloudprovider: {
+          template: "${file(\"openstack.conf\")}",
```
Comment (shashidharatd) [inline phase1/openstack/openstack.jsonnet]: @ss22ever, I am little curious where is this template file rendered? How is the `openstack.conf` file created in target VM's filesystem and how the cloud-provider is initialised with this config file.

File: phase1/openstack/openstack.jsonnet [hunk 2]
```diff
@@ -69,6 +69,17 @@ function(cfg)
             k8s_master_ip: "${openstack_compute_instance_v2.%(master_instance)s.network.0.fixed_ip_v4}" % names,
           },
         },
+       	//Populates openstack config file
```
Comment (shashidharatd) [inline phase1/openstack/openstack.jsonnet]: nit: alignment of the comment is slightly off.

**Comment by jessicaochen** [comment]
@ss22ever - Looks like this review has a LGTM but also has come comments to resolve before merge. Is that an accurate assessment of the situation?

**Pull request closed without merging**